examples/resources/grafana_slo/resource_update.tf

```diff
@@ -12,29 +12,28 @@ resource "grafana_slo" "update" {
     window = "7d"
   }
   label {
-    key   = "customkey"
-    value = "customvalue"
+    key   = "slo"
+    value = "terraform"
   }
   alerting {
     fastburn {
       annotation {
         key   = "name"
-        value = "Critical - SLO Burn Rate Alert"
+        value = "SLO Burn Rate Very High"
       }
-      label {
-        key   = "type"
-        value = "slo"
+      annotation {
```

Comment:
I see that you're adding a second annotation (you want `description` here I think, not `name` again), but why did you remove the label as well?

Most alerts will have name and description annotations (helps with notifications), but we also allow labels. Why not show both?